tests/kwlib_test.sh

```diff
@@ -392,4 +392,30 @@ function test_exit_msg()
   assertEquals "($LINENO) We expected 3 as a return" 3 "$ret"
 }
 
+function test_kw_parse()
+{
+  local lo='xpto:,foo,bar'
+  local so='x:,f,b'
+  local out
+  local expected
+
+  out="$(kw_parse "$so" "$lo" --xpto 1 --foo --bar biz)"
+  expected=" --xpto '1' --foo --bar -- 'biz'"
+  assertEquals "($LINENO)" "$expected" "$out"
+}
+
+function test_kw_parse_get_errors()
+{
+  local lo='xpto:,foo,bar'
+  local so='x:,f,b'
+  local out
+  local expected
+
+  out="$(kw_parse_get_errors "$so" "$lo" --fee --bar biz --xpto)"
+  expected=(
+    "kw: unrecognized option '--fee'" "kw: option '--xpto' requires an argument"
+  )
+  compare_command_sequence expected[@] "$out" "$LINENO"
+}
```

Comment:
Idk why, but when I tried this test locally, it failed, and based on the way that you're calling `kw_parse_get_errors`, I actually expect a failure... the weird thing is why CI did not complain about it. Is this something specific to my setup? Btw, I fixed it by using:

```diff
diff --git a/tests/kwlib_test.sh b/tests/kwlib_test.sh
index 2a82ec8..9d95a4c 100755
--- a/tests/kwlib_test.sh
+++ b/tests/kwlib_test.sh
@@ -406,16 +406,17 @@ function test_kw_parse()
 
 function test_kw_parse_get_errors()
 {
-  local lo='xpto:,foo,bar'
-  local so='x:,f,b'
-  local out
-  local expected
+  local long_option='xpto:,foo,bar'
+  local short_option='x:,f,b'
+  local output
 
-  out="$(kw_parse_get_errors "$so" "$lo" --fee --bar biz --xpto)"
-  expected=(
-    "kw: unrecognized option '--fee'" "kw: option '--xpto' requires an argument"
+  declare -a expected_errors=(
+    "kw: unrecognized option '--fee'"
+    "kw: option '--xpto' requires an argument"
   )
-  compare_command_sequence expected[@] "$out" "$LINENO"
+
+  output=$(kw_parse_get_errors 'kw' "$short_option" "$long_option" --fee --bar biz --xpto)
+  compare_command_sequence expected_errors[@] "$output" "$LINENO"
 }
 
 invoke_shunit
```

From Github check, I think something is wrong:
https://github.com/kworkflow/kworkflow/pull/390/checks?check_run_id=3302532172#step:5:320

Could you check that?

Reply:
I think I added the option to pass a name to kw_parse_get_errors but forgot to add it to the tests!